### ISSUE TITLE
Top bar disappears quickly after page load, impossible to click "Follow"

### ISSUE THREAD

**Issue by hmemcpy** [sindresorhus/refined-twitter#135]
See this gif:
![aug-16-2018 20-00-36](https://user-images.githubusercontent.com/601206/44223074-3a946880-a18f-11e8-8568-556e34898f8e.gif)

I'd really like to keep this bar permanently visible to see followers count on any profile.

**Comment by jorgegonzalez** [comment]
Hmm I can't reproduce.

**Comment by sindresorhus** [comment]
I can reproduce this every time on Chrome on Mac, for example: https://twitter.com/sindresorhus

**Comment by jorgegonzalez** [comment]
Weird, also on Chrome for Mac and I'm not seeing this issue.

**Comment by azdanov** [comment]
Same but on Windows 10.
It happens when screen height gets larger than 720px.

![twitter com_sindresorhus](https://user-images.githubusercontent.com/6123841/44307817-d00f4280-a3b1-11e8-9a22-fe0349b2e735.png)

**Comment by jorgegonzalez** [comment]
Thanks @azdanov I can reproduce.

**Comment by OleksandrBilenko** [comment]
Try this solution: https://github.com/sindresorhus/refined-twitter/issues/129#issuecomment-415083297

**Comment by ryota-murakami** [comment]
Yeah, surely that will be fix if you turn off `Hide Profile Card`.(https://github.com/sindresorhus/refined-twitter/issues/135#issuecomment-415083466)

But problem is **also hide follow button toolbar** when you turn of `Hide Profile Card`.
As @azdanov says button toolbar is  shown if you enable chrome devtools.
So i think might be ideal behavior is **hide profile card only,  button toolbar`**

**Comment by ryota-murakami** [comment]
I think might be the issue is relative to `Window.innerHeight`.
I tried change `Window.innerHeight` value with Chrome DevTools.

Please refer following video, if DevTools lift up, follow button was showing.
Else if DevTools lift down, follow button was hiding.

## Video
https://www.dropbox.com/s/r9ey72dkah2wprt/tweetButton.mp4?dl=0

**Comment by faahim** [comment]
I'm having the same issue! After having it for a few days, I decided to create an issue and recorded this video for reference. Good to see it's already brought up! It's been really hard to follow people! :slightly_smiling_face: 

[Video Here](https://streamable.com/e7eg8)

I use Arch Linux and a 1080p screen, just in case it helps.
